### PR TITLE
citra_android: Implement edge-to-edge

### PR DESCRIPTION
**Pull request by t895**
The app UI now reaches behind the system bars.

[Screencast from 03-14-2023 02:51:05 PM.webm](https://user-images.githubusercontent.com/14132249/225108116-703b2248-bfa7-4442-97a0-097033561a37.webm)

Side note - The more I look at the state of the dark theme, the more unhappy I am. Expect an update for that soon too.

- - -
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/6349)

### CODE REVIEW

File: src/android/app/src/main/java/org/citra/citra_emu/features/cheats/ui/CheatListFragment.java [hunk 1]
```diff
@@ -19,6 +22,9 @@
 import org.citra.citra_emu.ui.DividerItemDecoration;
 
 public class CheatListFragment extends Fragment {
+    private RecyclerView mRecyclerView;
```
Comment (FearlessTobi) [inline src/android/app/src/main/java/org/citra/citra_emu/features/cheats/ui/CheatListFragment.java:25]: nit: we don't use the mXXX naming scheme

Comment (t895) [reply src/android/app/src/main/java/org/citra/citra_emu/features/cheats/ui/CheatListFragment.java:25]: You do?
https://github.com/citra-emu/citra/blob/6fbc54b0c5178dfb6aa05278c80a43d08445ec5f/src/android/app/src/main/java/org/citra/citra_emu/ui/main/MainActivity.java#L45

Comment (FearlessTobi) [reply src/android/app/src/main/java/org/citra/citra_emu/features/cheats/ui/CheatListFragment.java:25]: welp, seems like we got lazy when porting from Dolphin.
feel free to leave it then.

**Comment by SachinVin** [comment]
Please rebase

**Comment by t895** [comment]
Done

**Comment by SachinVin** [comment]
Thanks

**Review by FearlessTobi — APPROVED** [review]

**Comment by FearlessTobi** [comment]
Please rebase again

**Pull request merged**